sync/class.jetpack-sync-defaults.php

```diff
@@ -222,6 +223,7 @@ class Jetpack_Sync_Defaults {
 		'wprss_last_update_items',
 		'wp_automatic_cache',
 		'snapTW',
+		'data', // securpress - bummer to blacklist something so general, but necessary
```

Comment:
Minor issue - The comment mentions `securpress`. I think we meant `secupress` (without the `r` after `u`.